src/ec/packages/ec-component-card/card.story.js

```diff
@@ -2,62 +2,205 @@
 import { storiesOf } from '@storybook/html';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
 import withCode from '@ecl-twig/storybook-addon-code';
+import { withKnobs, text, array } from '@storybook/addon-knobs';
+import merge from 'deepmerge';
 
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import dataCard from './demo/data--card';
-import dataCardEvent from './demo/data--card-event';
-import dataCardTag from './demo/data--card-tag';
 import dataTile from './demo/data--tile';
+import dataCardTag from './demo/data--card-tag';
+import dataCardEvent from './demo/data--card-event';
 
 import card from './ecl-card.html.twig';
 import notes from './README.md';
 
-const formatInfo = data => {
-  if (data.card.infos) {
-    data.card.infos.forEach(item => {
-      item.icon.path = defaultSprite;
-    });
-  }
+function prepareMerge(oldArrayName, newArrayName, label) {
+  oldArrayName.forEach((a, key) => {
+    if (label === 'label') {
+      newArrayName.push(oldArrayName[key].label);
+    } else {
+      newArrayName.push(oldArrayName[key]);
+    }
+  });
+  oldArrayName.splice(0, oldArrayName.length);
+}
+
+const dataCardInfos = [];
+const dataCardMeta = [];
+const dataCardTags = [];
+const dataTileLinks = [];
+const dataCardTagMeta = [];
+const dataCardTagTag = [];
+const dataCardEventInfos = [];
 
-  return data;
-};
+prepareMerge(dataCard.card.infos, dataCardInfos, 'label');
+prepareMerge(dataCard.card.meta, dataCardMeta);
+prepareMerge(dataCard.card.tags, dataCardTags, 'label');
+prepareMerge(dataTile.card.links, dataTileLinks, 'label');
+prepareMerge(dataTile.card.links, dataTileLinks, 'label');
+prepareMerge(dataCardTag.card.tags, dataCardTagTag, 'label');
+prepareMerge(dataCardTag.card.meta, dataCardTagMeta);
+prepareMerge(dataCardTag.card.tags, dataCardTagTag, 'label');
+prepareMerge(dataCardEvent.card.infos, dataCardEventInfos, 'label');
 
 storiesOf('Components/Card', module)
   .addDecorator(withCode)
+  .addDecorator(withKnobs)
   .addDecorator(withNotes)
   .add(
     'card',
-    () => {
-      return card(formatInfo(dataCard));
-    },
+    () =>
+      card(
+        merge(dataCard, {
+          card: {
+            title: {
+              label: text('Title card', dataCard.card.title.label),
+            },
+            description: text('Description', dataCard.card.description),
+            image: {
+              src: text('Image path', dataCard.card.image.src),
+            },
+            meta: array('Metatags', dataCardMeta, '|'),
+            infos: [
+              {
+                icon: {
+                  size: 'xs',
```

Comment:
I would avoid "hardcoding" these information here in the story, we should rely on the specs. I understand that the use case of a merge here is not the easiest ever but as said in the previous comment there is no need for an overall merge if that is overcomplicating things, what we want to have is knobs using default values coming from the specs, basically.